src/data-types/varbinary.ts

```diff
@@ -56,14 +56,42 @@ const VarBinary: { maximumLength: number } & DataType = {
   },
 
   generate: function* (parameter, options) {
-    if (parameter.value != null) {
-      const buffer = new WritableTrackingBuffer(0);
+    let value = parameter.value;
+
+    if (value != null) {
       if (parameter.length! <= this.maximumLength) {
-        buffer.writeUsVarbyte(parameter.value);
-      } else {
-        buffer.writePLPBody(parameter.value);
+        const buffer = new WritableTrackingBuffer(0);
+        buffer.writeUsVarbyte(value);
+        yield buffer.data;
+
+      } else { // writePLPBody
+        const UNKNOWN_PLP_LEN = Buffer.from([0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
```

Comment:
What do you think about making this a top-level constant? This value never changes, and that way we could save ourselves from creating this `Buffer` over and over again for each `VarBinary` parameter.